### PR TITLE
synchronize outbox semantics for params / events to use "merklePath" 

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: packages/arb-bridge-eth/contracts/bridge/Outbox.sol [hunk 1]
```diff
@@ -134,9 +134,9 @@ contract Outbox is IOutbox, Cloneable {
      * @notice Executes a messages in an Outbox entry.
      * @dev Reverts if dispute period hasn't expired, since the outbox entry
      * is only created once the rollup confirms the respective assertion.
-     * @param batchNum Index of OutboxEntry in outboxEntries array
+     * @param batchNum Number of target OutboxEntry
```
Comment (fredlacs) [inline packages/arb-bridge-eth/contracts/bridge/Outbox.sol:137]: Isn't this actually the `Index of OutboxEntry in outboxEntries array` as the previous doc?
I don't think `Number of target OutboxEntry` is clearer.

Maybe we can call it the identifier of the batch?

**Review by fredlacs — DISMISSED** [review]
LGTM just not sure about one natspec change

File: packages/arb-bridge-eth/contracts/bridge/Outbox.sol (hunk 1 above)
Comment (DZGoldman) [reply packages/arb-bridge-eth/contracts/bridge/Outbox.sol:137]: yeah not sure why i did this tbh

**Review by fredlacs — APPROVED** [review]
LGTM

**Comment by fredlacs** [comment]
> (I force-pushed b/c it was the easiest way to handle the old abis commit don't be mad)

I am mad, but just a bit.

**Pull request closed without merging**